src/naming.js

```diff
@@ -23,6 +23,24 @@ class Naming {
     return `${_.upperFirst(metricName)}${functionName}`;
   }
 
+  getDimensionsList(dimensionsList, funcRef) {
+    let funcNameDimension =  {
+      'Name': 'FunctionName',
+      'Value': {
+        Ref: funcRef
+      }
+    };
+    if(dimensionsList == null) {
+      return [funcNameDimension];
+    }
+    let filteredDimensions = dimensionsList.filter( (dim) => {
+      return dim.Name != 'FunctionName'
+    })
+    filteredDimensions.push(funcNameDimension);
+    console.log(filteredDimensions)
```

Comment:
@jandaluz can you please remove the `console.log` ?

Reply:
i've pushed a new commit removing the log statement